### PR TITLE
add trada

### PR DESCRIPTION
**Pull request by ocefpaf**
Redoing https://github.com/conda-forge/staged-recipes/pull/25531

Ping @doutriaux1 and @olson59.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please lint

File: recipes/trata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/LLNL/trata/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e94cc7a302c203c1654d987fc9a8e453cf45a95e659366911e4dfde84b1ace83
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  noarch: python
```
Comment (ocefpaf) [inline recipes/trata/meta.yaml:15]:
```suggestion
  noarch: python
  entry_points:
    - trata = trata.trata:__init__
```

Comment (ocefpaf) [reply recipes/trata/meta.yaml:15]: @doutriaux1 and @olson59 it was the lack of entrypoints that caused  the Windows failures. It should be fine now.

Comment (doutriaux1) [reply recipes/trata/meta.yaml:15]: Thanks @ocefpaf !

**Pull request merged**